Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -73,7 +73,7 @@
           },
           {
             "name": "Builds",
-            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds",
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any",
```

Comment:
@pascalberger I am not sure about what default branch depicts in teamcity configurations. Are you saying it is better to keep the existing behavior of picking up latest build from default branch or do you mean that we should have an option of default version at the time of linking for the extension to be usable in autotrigger scenario?